xExtension-ImageProxy/i18n/fr/ext.php

```diff
@@ -5,7 +5,7 @@
 		'proxy_url' => 'URL du proxy',
 		'scheme_http' => 'Proxy HTTP',
 		'scheme_https' => 'Proxy HTTPS',
-		'scheme_default' => 'Proxy indéterminé',
+		'scheme_default' => 'Proxy URLs relatifs au protocole',
```

Comment:
```suggestion
		'scheme_default' => 'Proxy URLs sans protocole',
```

I think that would be either to understand (*scheme* - as in URL - is typically called *protocole* in French)

Reply:
For reference (except I thought URLs were masculine).

https://github.com/hail2u/html-best-practices/blob/main/README.fr.md
https://phoenixnap.fr/glossaire/qu%27est-ce-qu%27une-url-relative
https://learn.microsoft.com/fr-fr/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/inline-live-issue-analysis

```suggestion
		'scheme_default' => 'Proxy URLs relatives au protocole',
```

Reply:
Yes yes, I am not saying that this is wrong, just likely to be harder to understand, but I do not really mind

Reply:
It's fine. Anyway, I think it's done now.